ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/arp.j2

```diff
@@ -8,3 +8,21 @@
 !
 arp aging timeout default {{ arp.aging.timeout_default }}
 {% endif %}
+{% if arp.static_entries is arista.avd.defined %}
+{# HACK - remove when dropping support for ansible 2.12.6 and its jinja2 hack #}
+{%     for entry in arp.static_entries %}
+{%         if entry.vrf is not arista.avd.defined %}
+{%             do entry.update({"vrf": "default"}) %}
```

Comment:
modifying the input data should be avoided. We do it elsewhere and we will drop 2.12.6 soon, so we can keep this for a short while.